btree/btree.c

```diff
@@ -2285,27 +2337,14 @@ static void generic_move(struct nd *src, struct nd *tgt,
 
 /** Insert operation section start point: */
 
-#ifndef __KERNEL__
+#if 0
 static bool m0_btree_overflow_is_possible(const struct nd *node, int ksize,
```

Comment:
Will it be needed later? If not, please remove from code.

Reply:
done